src/components/Form.js

```diff
@@ -0,0 +1,156 @@
+import { useState } from 'react';
+import axios from 'axios';
+import formQuestions from '../helpers/questions';
+
+const Form = () => {
+    //saving the API data in state
+  const [genre, setGenre] = useState("");
+
+
+  const [questions, setQuestions] = useState(formQuestions);
+ 
+//   const [error, setError] = useState(false);
```

Comment:
Hey, don't see this state used anywhere else so not sure if you meant to leave this on for later?